cmd/artifact.go

```diff
@@ -0,0 +1,90 @@
+package cmd
+
+import (
+	"encoding/json"
+	"fmt"
+
+	"github.com/charmbracelet/glamour"
+	"github.com/charmbracelet/lipgloss"
+	"github.com/massdriver-cloud/mass/internal/api"
+	"github.com/massdriver-cloud/mass/internal/config"
+	"github.com/spf13/cobra"
+)
+
+var artifactCommandHelp = mustRenderHelpDoc("artifact")
+
+var artifactCommand = &cobra.Command{
+	Use:   "artifact",
+	Short: "Manage Massdriver artifacts",
+	Long:  artifactCommandHelp,
+}
+
+var listArtifactsCommand = &cobra.Command{
+	Use:     "list",
+	Aliases: []string{"l", "ls"},
+	Short:   "List all artifacts",
+	RunE:    runListArtifacts,
+}
+
+var artifactInfoCommand = &cobra.Command{
+	Use:     "info <artifact id>",
+	Aliases: []string{"i"},
+	Short:   "Get information about an artifact",
+	Args:    cobra.ExactArgs(1),
+	RunE:    runArtifactInfo,
+}
+
+func init() {
+	rootCmd.AddCommand(artifactCommand)
+	artifactCommand.AddCommand(listArtifactsCommand)
+	artifactCommand.AddCommand(artifactInfoCommand)
+}
+
+func runListArtifacts(cmd *cobra.Command, args []string) error {
+	c := config.Get()
+	client := api.NewClient(c.URL, c.APIKey)
+	artifacts, err := api.GetAllArtifacts(client, c.OrgID)
+
+	if err != nil {
+		return err
+	}
+
+	fmt.Println("Artifacts:")
```

Comment:
Use a bubble tea table for presenting a table of items

https://github.com/Evertras/bubble-table